### PR TITLE
change to bl- alternatives where necessary

### PR DESCRIPTION
**Pull request by capn-damo**
...also changed tabs to spaces

### CODE REVIEW

**Comment by capn-damo** [comment]
@johnraff et al

I won't merge this yet until you approve :)

**Comment by johnraff** [comment]
I left some inline comments here and there.

**Comment by capn-damo** [comment]
As suggested:
Changed USR_CFG_DIR; test for images dir; add bl-tint2-pipemenu

File: bl-kb-pipemenu [hunk 1]
```diff
@@ -10,55 +10,56 @@
...
+    cp "$RCPATH" "$RCBKP"
+    bl-text-editor "$RCPATH"        # open rc.xml in default editor
 
 fi
 
 # pipemenu
 menuStart
-menuItem "Backup and Edit rc.xml" "bl-kb-pipemenu --rcedit"
-menuItem "Refesh keybinds list" "bl-kb.py" # use if rc.xml has been edited
+menuItem "Backup and Edit rc.xml" "$@ --rcedit"
```
Comment (johnraff) [inline bl-kb-pipemenu:45]: Is this command OK? Maybe should be $0?

Comment (capn-damo) [inline bl-kb-pipemenu:45]: You are correct - my bad :(

**Pull request merged**